Paymetheus.Rpc/Portability.cs

```diff
@@ -60,5 +60,26 @@ private static string ToDotLower(string value)
 
             return "." + value.ToLower();
         }
+
+        public static string ExecutableInstallationPath(PlatformID platform, string organization, string executableName)
+        {
+            // Unix and Mac: Unsure where the expected installation paths are or would be.
+            // These are staying unimplemeted until it actually matters.
+            switch (platform)
+            {
+                case PlatformID.Win32NT:
+                    return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles),
+                        ToUpper(organization), ToUpper(executableName), executableName.ToLower() + ".exe");
```

Comment:
Yeah that is no longer the reality in the msi. Per irc dicussions I moved dcrwallet.exe into the same directory as paymetheus.exe.